### PR TITLE
Bookstore: connect to API

### PR DESCRIPTION
**Pull request by tchasinga**
In the previous project, you added the add book and remove book features directly in your application. Now you will refactor your code to use an external API to facilitate these features.

Make use of the [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) as your backend.
Use [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) to make your HTTP requests to the backend
Fetch your books from the Bookstore API after the component has rendered
Refactor your add book and remove book features to persist your changes in the server
Your application should behave the same way after the refactoring

### CODE REVIEW

**Review by Ibizugbe — COMMENTED** [review]
Hi @tchasinga 👋🏼👋🏼👋🏼, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have webhint and lighthouse setup. Kindly set these linters in your project as they are required to check that you write better and more professional codes.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Ibizugbe** [comment]
@tchasinga , sorry about the invalidation, that was my bad. you can now proceed to submit👍🏼.

File: src/App.js [hunk 1]
```diff
@@ -1,24 +1,23 @@
-import React from 'react';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import { Provider } from 'react-redux';
-import Store from './redux/Storage';
-import BookList from './Components/BookList';
-import Navigation from './Components/Navigation';
-import Categories from './Components/categories';
+import Book from 'Components/Book';
+import Categories from 'Components/Categories';
```
Comment (julie-ify) [inline src/App.js:3]: - [ ] You have an import error. Kindly fix that.

  image

  Should be this:

  ```
  import Categories from 'Components/categories';
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @tchasinga,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @tchasinga  👋 

### [OPTIONAL]
- [ ] It's really confusing to understand what flow you're using until I've to manually check it from your repository. Please rename your development branch name to something like dev / develop / development. mainDev branch name seems really confusing. 👍 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**